### DISCUSSION TITLE
Issues Editing Footer Button Redirect Location

### DISCUSSION THREAD

**Discussion by jackson979** [flyntwp/flynt#445] [Q&A]
I am attempting to edit the footer of my WordPress website to change where a button redirects to. When I try to edit it through the normal Page editing, it only shows a box to edit the button text, not where it redirects to, as it normally does. Does anyone know where I might find that setting? I posted on the WordPress forum and they said to look into the theme I am using (Flynt). Any insight would be greatly appreciated.

**Comment by timohubois** [comment]
@jackson979 did you mean the component [Navigation Footer](https://github.com/flyntwp/flynt/tree/master/Components/NavigationFooter)? If yes, you can find relevant settings on the "Translatable Options" page at the backend. If no, please provide more details about what exactly you want change where.

**Reply by jackson979** [reply]
I do not think it is tied to the Navigation Footer, but I could be wrong. When I look at the Navigation Footer settings in the "Translatable Options" page, there is nothing in the Content box. 

On the current active website, there is a button in the footer that redirects to another page. When I go to my dashboard and go to Pages>Global Footer (name of the page)>Edit, there is a spot that says "Button Text" with an input field, but no input field telling the button where to go. I assume that the theme has that link referenced somewhere, but I cannot for the life of me find where that may be. I posted on the WordPress forums and they said to check with the theme developer.

**Reply by timohubois** [reply]
The input is not related to the theme (if you have not changed any code). If you search inside the entire repository of flynt, you cannot find the term "Button Text" without any context: https://github.com/flyntwp/flynt/search?q=%22Button+Text%22

Anyway, I think it might can be a custom field from the plugin Advanced Custom Fields Pro. A screenshot would be helpful.

**Reply by timohubois** [reply]
@jackson979 did you solve the issue?
